# Kashubian translation for linuxmint
# Copyright (c) 2009 Rosetta Contributors and Canonical Ltd 2009
# This file is distributed under the same license as the linuxmint package.
# FIRST AUTHOR <EMAIL@ADDRESS>, 2009.
#
msgid ""
msgstr ""
"Project-Id-Version: linuxmint\n"
"Report-Msgid-Bugs-To: FULL NAME <EMAIL@ADDRESS>\n"
"POT-Creation-Date: 2014-04-14 12:43+0100\n"
"PO-Revision-Date: 2010-05-19 09:21+0000\n"
"Last-Translator: Launchpad Translations Administrators <Unknown>\n"
"Language-Team: Kashubian <csb@li.org>\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"X-Launchpad-Export-Date: 2015-06-02 10:09+0000\n"
"X-Generator: Launchpad (build 17532)\n"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:34
#: usr/lib/linuxmint/mintBackup/mintBackup.py:99
#: usr/lib/linuxmint/mintBackup/mintBackup.py:238
#: usr/lib/linuxmint/mintBackup/mintBackup.py:427
#: usr/lib/linuxmint/mintBackup/mintBackup.py:443
#: usr/lib/linuxmint/mintBackup/mintBackup.py:501
#: usr/lib/linuxmint/mintBackup/mintBackup.py:504
#: usr/lib/linuxmint/mintBackup/mintBackup.py:556
#: usr/lib/linuxmint/mintBackup/mintBackup.py:559
#: usr/lib/linuxmint/mintBackup/mintBackup.py:579
#: usr/lib/linuxmint/mintBackup/mintBackup.py:602
#: usr/lib/linuxmint/mintBackup/mintBackup.py:609
#: usr/lib/linuxmint/mintBackup/mintBackup.py:1426
#: usr/lib/linuxmint/mintBackup/mintBackup.py:1519
#: usr/lib/linuxmint/mintBackup/mintBackup.py:1527
#: usr/lib/linuxmint/mintBackup/mintBackup.py:1570
msgid "Backup Tool"
msgstr "Nôrzãdze zazychrowaniô pòdôwków"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:35
msgid "Make a backup of your home directory"
msgstr "Zazychrëjë pòdôwczi twòjegò domôcegò kataloga"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:155
msgid "Preserve structure"
msgstr "Ùtrzëmôj strukturã"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:157
msgid ".tar file"
msgstr "lopk .tar"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:158
msgid ".tar.bz2 file"
msgstr "lopk .tar.bz2"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:159
msgid ".tar.gz file"
msgstr "lopk .tar.gz"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:165
msgid "Never"
msgstr "Nigdë"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:166
msgid "Size mismatch"
msgstr "Lëchô miara"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:167
msgid "Modification time mismatch"
msgstr "Czas mòdifikacëji nie pasëje"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:168
msgid "Checksum mismatch"
msgstr "Kòntrolowóne zesëmòwanié nie pasëje"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:169
msgid "Always"
msgstr "Wiedno"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:191
msgid "Excluded paths"
msgstr "Wëłączoné stegnë"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:201
msgid "Type"
msgstr "Ôrt"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:205
msgid "Detail"
msgstr "Detale"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:211
#: usr/lib/linuxmint/mintBackup/mintBackup.py:220
msgid "Path"
msgstr "Stegna"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:214
#: usr/lib/linuxmint/mintBackup/mintBackup.py:223
msgid "Error"
msgstr "Fela"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:255
msgid "Store?"
msgstr "Ùtrzëmac?"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:258
#: usr/lib/linuxmint/mintBackup/mintBackup.py:271
msgid "Name"
msgstr "Miono"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:268
msgid "Install"
msgstr "Instalëjë"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:276
msgid "Backup or Restore"
msgstr "Kòpijô bezpiekù ë doprowôdzanié nazôd"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:277
msgid "Choose from the following options"
msgstr "Wëbierzë jedną z nôslédnych òptacëjów"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:279
#: usr/lib/linuxmint/mintBackup/mintBackup.py:295
#: usr/lib/linuxmint/mintBackup/mintBackup.py:309
#: usr/lib/linuxmint/mintBackup/mintBackup.py:316
#: usr/lib/linuxmint/mintBackup/mintBackup.py:320
#: usr/lib/linuxmint/mintBackup/mintBackup.py:325
msgid "Backup files"
msgstr "Kòpijô bezpiekù lopków"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:280
#: usr/lib/linuxmint/mintBackup/mintBackup.py:329
#: usr/lib/linuxmint/mintBackup/mintBackup.py:339
#: usr/lib/linuxmint/mintBackup/mintBackup.py:345
#: usr/lib/linuxmint/mintBackup/mintBackup.py:350
msgid "Restore files"
msgstr "Doprowôdzë lopczi nazôd"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:281
#: usr/lib/linuxmint/mintBackup/mintBackup.py:354
#: usr/lib/linuxmint/mintBackup/mintBackup.py:359
#: usr/lib/linuxmint/mintBackup/mintBackup.py:365
#: usr/lib/linuxmint/mintBackup/mintBackup.py:370
msgid "Backup software selection"
msgstr "Ùsôdzë kòpijã bezpiekù wëbróny softwôrë"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:282
#: usr/lib/linuxmint/mintBackup/mintBackup.py:374
#: usr/lib/linuxmint/mintBackup/mintBackup.py:379
#: usr/lib/linuxmint/mintBackup/mintBackup.py:386
msgid "Restore software selection"
msgstr "Doprowôdzë wëbróną softwôrã nazôd"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:284
msgid "Make a backup of your files"
msgstr "Ùsôdzë kòpiją bezpiekù swòjich lopków"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:285
msgid "Save the list of installed applications"
msgstr "Zapiszë lëstã winstalowónych aplikacëjów"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:286
msgid "Restore a previous backup"
msgstr "Doprowôdzë pòprzédną kòpijã bezpiekù nazôd"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:287
msgid "Restore previously installed applications"
msgstr "Doprowôdzë pòprzédno winstalowóné aplikacëje nazôd"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:296
msgid "Please select a source and a destination for your backup"
msgstr "Wëbierzë proszã zdrój abò plac dlô twòji Kòpiji bezpiekù"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:297
#: usr/lib/linuxmint/mintBackup/mintBackup.py:333
#: usr/lib/linuxmint/mintBackup/mintBackup.py:341
msgid "Source:"
msgstr "Zdrój:"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:298
#: usr/lib/linuxmint/mintBackup/mintBackup.py:334
msgid "Destination:"
msgstr "Cél:"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:299
#: usr/lib/linuxmint/mintBackup/mintBackup.py:335
msgid "Advanced options"
msgstr "Awansowóné òptacëje"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:300
#: usr/lib/linuxmint/mintBackup/mintBackup.py:342
msgid "Description:"
msgstr "Òpisënk:"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:301
msgid "Output:"
msgstr "Wińdzenié:"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:302
#: usr/lib/linuxmint/mintBackup/mintBackup.py:336
msgid "Overwrite:"
msgstr "Nadpisëwanié:"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:303
msgid "Confirm integrity"
msgstr "Pòcwierdzë integralnotã"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:304
msgid "Follow symlinks"
msgstr "Biéj za symbòlicznyma lënkama"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:305
msgid "Preserve permissions"
msgstr "Ùtrzëmôj prawa przistãpù"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:306
msgid "Preserve timestamps"
msgstr "Ùtrzëmôj znakòwniczi czasu"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:310
msgid "Please select any files or directories you want to exclude"
msgstr ""
"Wëbierzë proszã swòjowólné lopczi abò katalodżi, chtërné chcesz wëkluczëc"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:311
msgid "Exclude files"
msgstr "Wëłączoné lopczi"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:312
msgid "Exclude directories"
msgstr "Wëkluczë katalodżi"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:313
msgid "Remove"
msgstr "Rëmôj"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:317
msgid "Please review the information below before starting the backup"
msgstr ""
"Przezdrzi proszã ùniższą wëdowiédzã, niglë naczniesz ùsôdzac kòpijã bezpiekù"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:321
msgid "Please wait while your files are being backed up"
msgstr "Proszã żdac przë ùsôdzaniu kòpiji bezpiekù twòjich lopków"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:322
#: usr/lib/linuxmint/mintBackup/mintBackup.py:367
msgid "Backing up:"
msgstr "Ùsôdzanié kòpiji bezpiekù:"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:326
#: usr/lib/linuxmint/mintBackup/mintBackup.py:371
msgid "The backup is now finished"
msgstr "Ùsôdzanié kòpiji bezpiekù ùdało zakùńczoné"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:330
msgid ""
"Please choose the type of backup to restore, its location and a destination"
msgstr ""
"Wëbierzë proszã ôrt kòpiji bezpiekù, chtërną chcesz doprowôdzëc nazôd, ji "
"pòłożenié ë przeznaczony plac"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:331
msgid "Archive"
msgstr "Archiwa"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:332
msgid "Directory"
msgstr "Katalog"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:340
msgid "Please review the information below"
msgstr "Przezdrzi proszã ùniższą wëdowiédzã"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:346
msgid "Please wait while your files are being restored"
msgstr "Proszã żdac przë doprowôdzaniu twòjich lopków nazôd"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:347
msgid "Restoring:"
msgstr "Doprowôdzanié nazôd:"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:351
msgid "The restoration of the files is now finished"
msgstr "Doprowôdzanié lopków nazôd ùdało zakùńczoné"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:355
msgid "Please choose a destination"
msgstr "Proszã wëbrac cél"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:356
#: usr/lib/linuxmint/mintBackup/mintBackup.py:520
msgid "Destination"
msgstr "Cél"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:360
msgid "The list below shows the packages you added to Linux Mint"
msgstr "Ùniższô lësta wëskrzëniwô paczétë, chtërné môsz dodóné do Linux Mint"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:361
#: usr/lib/linuxmint/mintBackup/mintBackup.py:381
msgid "Select all"
msgstr "Wëbierzë wszëtkò"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:362
#: usr/lib/linuxmint/mintBackup/mintBackup.py:382
msgid "Deselect all"
msgstr "Òdznaczë wszëtkò"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:366
msgid "Please wait while your software selection is being backed up"
msgstr "Proszã żdac, jak je ùsôdzanô kòpijô bezpiekù wëbróny softwôrë"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:375
msgid "Please select a saved software selection"
msgstr "Wëbierzë proszã zapisôny wëbiérk softwôrë"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:376
msgid "Software selection:"
msgstr "Wëbiérk softwôrë:"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:380
msgid "Select the packages you want to install"
msgstr "Wëbierzë paczétë, chtërné chcesz wistalowac"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:383
msgid "Refresh"
msgstr "Zladëjë znowa"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:387
msgid "The restoration is now finished"
msgstr "Zakùńczono doprôwdzanié nazôd"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:388
msgid "Your package selection was restored succesfully"
msgstr "Twòje paczétë òstałe domëslno doprowôdzoné nazôd"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:436
#: usr/lib/linuxmint/mintBackup/mintBackup.py:452
msgid "Invalid path"
msgstr "Lëchô stegna"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:436
#: usr/lib/linuxmint/mintBackup/mintBackup.py:452
#, python-format
msgid "%s is not located within your source directory. Not added."
msgstr "%s nie je w Twòjim zdrojowim katalogù. Nie òstôł dodóny."

#: usr/lib/linuxmint/mintBackup/mintBackup.py:501
msgid "Please choose directories for the source and the destination"
msgstr ""

#: usr/lib/linuxmint/mintBackup/mintBackup.py:504
msgid ""
"Please choose different directories for the source and the destination"
msgstr "Wëbierzë proszã wszelejaczé katalodżi dlô zdroju ë docélowégò placu"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:519
msgid "Source"
msgstr "Zdrój"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:522
msgid "Description"
msgstr "Òpisënk"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:526
msgid "Compression"
msgstr "Kòmpresëjô"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:530
msgid "Overwrite destination files"
msgstr "Nadpiszë docélowé lopczi"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:533
msgid "Exclude"
msgstr "Wëłączë"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:556
msgid "Please choose a file to restore from"
msgstr "Wëbierzë proszã lopk, z chtërnegò bãdzesz doprowôdzac nazôd"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:559
#: usr/lib/linuxmint/mintBackup/mintBackup.py:579
msgid "Please choose a destination directory"
msgstr "Wëbierzë proszã docélowi katalog"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:602
msgid "No packages need to be installed at this time"
msgstr "W tim czasu nie dô niżódnegò nótérnegò paczétu do winstalowaniô"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:609
msgid "Please select one or more packages to install"
msgstr "Wëbierzë proszã jeden abò wicy paczétów do winstalowaniô"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:666
#: usr/lib/linuxmint/mintBackup/mintBackup.py:667
#: usr/lib/linuxmint/mintBackup/mintBackup.py:1143
#: usr/lib/linuxmint/mintBackup/mintBackup.py:1145
#: usr/lib/linuxmint/mintBackup/mintBackup.py:1439
#: usr/lib/linuxmint/mintBackup/mintBackup.py:1440
msgid "Calculating..."
msgstr "Rechòwanié..."

#: usr/lib/linuxmint/mintBackup/mintBackup.py:716
#: usr/lib/linuxmint/mintBackup/mintBackup.py:756
msgid "Skipping broken link"
msgstr "Przepùszczanié lëchegò lënka"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:717
msgid "Broken link"
msgstr "Lëchi lënk"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:720
#: usr/lib/linuxmint/mintBackup/mintBackup.py:760
msgid "Skipping link"
msgstr "Przepùsczë lënk"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:784
#: usr/lib/linuxmint/mintBackup/mintBackup.py:793
#: usr/lib/linuxmint/mintBackup/mintBackup.py:802
#: usr/lib/linuxmint/mintBackup/mintBackup.py:1204
#: usr/lib/linuxmint/mintBackup/mintBackup.py:1215
#: usr/lib/linuxmint/mintBackup/mintBackup.py:1228
#: usr/lib/linuxmint/mintBackup/mintBackup.py:1293
#: usr/lib/linuxmint/mintBackup/mintBackup.py:1302
#: usr/lib/linuxmint/mintBackup/mintBackup.py:1312
msgid "Skipping identical file"
msgstr "Przepùszczanié równegò lopka"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:824
#, python-format
msgid ""
"Warning: Some files were not saved, copied: %(current_file)d files out of "
"%(total)d total"
msgstr ""
"Bôczënk: Niejedné lopczi nie òstałé zapisóné, skòpérowóno: %(current_file)d "
"lopków z całowny wielënë %(total)d"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:828
msgid "An error occured during the backup"
msgstr "Pòkôza sã fela przë ùsôdzaniu kòpiji bezpiekù"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:838
#: usr/lib/linuxmint/mintBackup/mintBackup.py:1484
msgid "The backup was aborted"
msgstr "Przerwóno ùsôdzanié kòpiji bezpiekù"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:846
msgid "The backup completed successfully"
msgstr "Ùdałé ùsôdzenié kòpiji bezpiekù"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:931
msgid "Checksum Mismatch:"
msgstr "Kòntrolné zesëmòwanié nie pasëje:"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:932
msgid "Checksum Mismatch"
msgstr "Kòntrolné zesëmòwanié nie pasëje"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:982
#: usr/lib/linuxmint/mintBackup/mintBackup.py:984
#: usr/lib/linuxmint/mintBackup/mintBackup.py:1011
msgid "Calculating checksum"
msgstr "Rechòwanié kòntrolnegò zesëmòwaniô"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:1053
#: usr/lib/linuxmint/mintBackup/mintBackup.py:1077
msgid "(Not created with the Backup Tool)"
msgstr "(Nie ùsôdzoné przez nôrzãdze kòpiji bezpiekù)"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:1336
#, python-format
msgid ""
"Warning: Some files were not restored, copied: %(current_file)d files out of "
"%(total)d total"
msgstr ""

#: usr/lib/linuxmint/mintBackup/mintBackup.py:1339
msgid "An error occured during the restoration"
msgstr "Pòkôza sã fela przë doprowôdzniu nazôd"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:1350
msgid "The restoration was aborted"
msgstr "Òprzestóné doprowôdzanié nazôd"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:1358
msgid "The restoration completed successfully"
msgstr "Ùdałé doprowôdzanié nazôd"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:1475
msgid "An error occured during the backup:"
msgstr "Pòkôza sã fela przë ùsôdzaniu kòpiji bezpiekù:"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:1492
msgid "Your software selection was backed up succesfully"
msgstr "Ùdałé ùsôdzanié kòpiji bezpiékù wëbróny softwôrë"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:1519
msgid "The specified file is not a valid software selection"
msgstr "Nacéchòwóny lopk nie je dobrim wëbierkã softwôrë"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:1527
#: usr/lib/linuxmint/mintBackup/mintBackup.py:1570
msgid "An error occurred while accessing the file"
msgstr "Pòkôza sã fela przistãpù do lopka"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:1563
msgid "Could not locate the package"
msgstr "Ni mòże nalezc paczétu"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:1594
msgid "Please wait, this can take some time"
msgstr "Proszã żdac, to mòże kąsk zbawic"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:1596
msgid "The installation is complete"
msgstr "Instalacëjô zakùńczonô"

#~ msgid "Archiving your home directory..."
#~ msgstr "Archiwizowanié twòjegò domôcegò kataloga..."

#~ msgid "Restoration successful"
#~ msgstr "Ùdałé doprowôdzanié nazôd"

#~ msgid "Cleaning up..."
#~ msgstr "Czëszczenié..."

#~ msgid "Restoration failed"
#~ msgstr "Doprowôdzanié nazôd nie darzëło sã"

#~ msgid "Read error"
#~ msgstr "Fela czëtaniô"

#~ msgid "Exclude folders"
#~ msgstr "Wëłączoné katalodżi"

#~ msgid "Hidden paths"
#~ msgstr "Zataconé stegnë"

#~ msgid "<big><b>Load data into your home directory</b></big>"
#~ msgstr "<big><b>Ladëjë pòdôwczi do domôcegò kataloga</b></big>"

#~ msgid "is not located within your home directory. Not added."
#~ msgstr "nie je w domôcym katalogù. Ni dodóné."

#~ msgid "Excluded Files and Directories"
#~ msgstr "Wëłączoné lopczi ë katalodżi"

#~ msgid "Restore"
#~ msgstr "Doprowôdzë nazôd"

#~ msgid "Overwrite existing files"
#~ msgstr "Nadpiszë lopczi"

#~ msgid "View content"
#~ msgstr "Òbôczë zamkłosc"

#~ msgid "Your home directory was successfully backed-up into"
#~ msgstr "Zazychrowanié pòdôwków domôcegò kataloga òsta ùdało zapisónô do"

#~ msgid "An error occurred during the backup:"
#~ msgstr "Fela przë ùsôdzaniu zazychrowaniô pòdôwków:"

#~ msgid "Backup failed"
#~ msgstr "Zazychrowanié pòdôwków ni òstało zapisóné"

#~ msgid "Extracting the content of the backup into your home directory..."
#~ msgstr ""
#~ "Rozpakòwëwanié zamkłoscë zazychrowaniô pòdôwków do domôcegò kataloga..."

#~ msgid "Copying the backup file into your home directory..."
#~ msgstr "Kòpérowanié lopka zazychrowaniô pòdôwków do domôcegò kataloga..."

#~ msgid "Backup successful"
#~ msgstr "Zazychrowanié pòdôwków ùdało zakùńczonôé"

#~ msgid "Your backup was successfully restored"
#~ msgstr "Zazychrowanié pòdôwków ùdało doprowôdzoné nazôd"

#~ msgid "An error occurred while restoring the backup archive:"
#~ msgstr "Fela przë doprowôdzaniu zazychrowaniô pòdôwków nazôd:"

#~ msgid "Opening the backup archive..."
#~ msgstr "Òtmëkanié archiwùm zazychrowaniô pòdôwków..."

#~ msgid "An error occurred while opening the backup:"
#~ msgstr "Fela przë òtmëkaniu zazychrowaniô pòdôwków:"

#~ msgid "Restore your personal data from this backup"
#~ msgstr "Doprowôdzë swòje pòdôwczi z negò zazychrowaniô nazôd"

#~ msgid "Included hidden directories"
#~ msgstr "Dołączoné zataconé katalodżi"

#~ msgid "Included"
#~ msgstr "Dołączoné"

#~ msgid "Backup"
#~ msgstr "Zazychrowanié pòdôwków (backup)"
